### PR TITLE
Add support to fetch a SLA by API call

### PR DESCRIPTION
**Pull request by izontm**
The code base is done, however I have some doubts about the expected output for the command. Fix issue #26

### CODE REVIEW

**Comment by aguimaraesduarte** [comment]
Hi @izontm thanks for the PR.
It looks good, but the unit tests for the new function is missing. Could you add them? [Here](https://github.com/globocom/azkaban-cli/tree/master/azkaban_cli/tests/test_azkaban) are the current tests that exist in the project, which you can use for inspiration :)

**Comment by aguimaraesduarte** [comment]
@izontm You should also update the README with the command you are adding ;)

**Comment by izontm** [comment]
My mistake, now I updated the README and did the tests.

File: azkaban_cli/azkaban_cli.py [hunk 1]
```diff
@@ -224,6 +225,15 @@ def __fetch_projects(ctx, user):
     except FetchProjectsError as e:
         logging.error(str(e))
 
+@login_required
+def __fetch_sla(ctx, schedule):
+    azkaban = ctx.obj[u'azkaban']
+
+    try:
+        data = azkaban.fetch_sla(schedule)
```
Comment (aguimaraesduarte) [inline azkaban_cli/azkaban_cli.py]: You're capturing the response, but not doing anything with it (like logging it to the console).

The response will be something like this when it succeeds:
```{'slaEmails': ['my.email@test.company.com'], 'allJobNames': ['job_1, 'job_2, 'job_3'], 'settings': [{'duration': '300m', 'rule': 'SUCCESS', 'id': 'test_id', 'actions': ['EMAIL']}]}```

I suggest you add a parsing function that will parse the response and print (`logging.info`) the output in a more human-readable fashion.

Take a look at [this PR here](https://github.com/globocom/azkaban-cli/pull/37/files#diff-4f86e3e923f34200240a9d0374b9c2c7), where I implement something very similar to what you want. Let me know if you have any questions

**Review by aguimaraesduarte — CHANGES_REQUESTED** [review]

**Review by aguimaraesduarte — APPROVED** [review]

**Comment by aguimaraesduarte** [comment]
Looks good to me! @izontm thanks for contributing!

**Pull request merged**